server/src/libs/validation.ts

```diff
@@ -45,4 +45,16 @@ export const ResetPasswordSchema = z.object({
   confirm_password: z.string({ message: "confirm password is require" }).min(6, "Password is required"),
 });
 
+export const NewCustomerSchema = z.object({
```

Comment:
please use camelCase for define variable

Reply:
We will use `Pascal`.